cluster/defaults.go

```diff
@@ -72,35 +72,52 @@ const (
-	DefaultAciSnatPortRangeStart       = "5000"
-	DefaultAciSnatPortRangeEnd         = "65000"
-	DefaultAciSnatPortsPerNode         = "3000"
-	DefaultAciUseHostNetnsVolume       = "false"
-	DefaultAciRunGbpContainer          = "false"
-	DefaultAciRunOpflexServerContainer = "false"
-	DefaultAciUseAciAnywhereCRD        = "false"
-	DefaultAciEnableEndpointSlice      = "false"
-	DefaultAciOpflexClientSSL          = "true"
-	DefaultAciUsePrivilegedContainer   = "false"
-	DefaultAciUseOpflexServerVolume    = "false"
+	DefaultAciApicRefreshTime                   = "1200"
+	DefaultAciOVSMemoryLimit                    = "1Gi"
+	DefaultAciImagePullPolicy                   = "Always"
+	DefaultAciImagePrefix                       = "noiro"
+	DefaultAciCniOperatorVersion                = "None"
+	DefaultAciServiceMonitorInterval            = "5"
+	DefaultAciPBRTrackingNonSnat                = "false"
+	DefaultAciInstallIstio                      = "false"
+	DefaultAciIstioProfile                      = "demo"
+	DefaultAciDropLogEnable                     = "true"
+	DefaultAciControllerLogLevel                = "info"
+	DefaultAciHostAgentLogLevel                 = "info"
+	DefaultAciOpflexAgentLogLevel               = "info"
+	DefaultAciUseAciCniPriorityClass            = "false"
+	DefaultAciNoPriorityClass                   = "false"
+	DefaultAciMaxNodesSvcGraph                  = "32"
+	DefaultAciSnatContractScope                 = "global"
+	DefaultAciSnatNamespace                     = "aci-containers-system"
+	DefaultAciDisablePeriodicSnatGlobalInfoSync = "false"
+	DefaultAciCApic                             = "false"
+	DefaultAciPodSubnetChunkSize                = "32"
+	DefaultAciSnatPortRangeStart                = "5000"
+	DefaultAciSnatPortRangeEnd                  = "65000"
+	DefaultAciSnatPortsPerNode                  = "3000"
+	DefaultAciUseHostNetnsVolume                = "false"
+	DefaultAciRunGbpContainer                   = "false"
+	DefaultAciRunOpflexServerContainer          = "false"
+	DefaultAciUseAciAnywhereCRD                 = "false"
+	DefaultAciEnableEndpointSlice               = "false"
+	DefaultAciOpflexClientSSL                   = "true"
+	DefaultAciUsePrivilegedContainer            = "false"
+	DefaultAciUseOpflexServerVolume             = "false"
+	DefaultApicSubscriptionDelay                = "None"
+	DefaultApicRefreshtickerAdjust              = "None"
+	DefaultOpflexDeviceDeleteTimeout            = "None"
+	DefaultEnableUpdates                        = "false"
+	DefaultMTUHeadroom                          = "None"
+	DefaultNodeSvcSubnet                        = "None"
+	DefaultDisableWaitForNetwork                = "false"
+	DefaultDurationWaitForNetwork               = "210"
+	DefaultPodSubnet                            = "None"
+	DefaultWatchNamespace                       = ""
+	DefaultUseAppsAPI                           = "apps/v1"
+	DefaultKubeConfigController                 = "1.1.1.1"
```

Comment:
What is the functionality of having this default to 1.1.1.1? I can see this causing issues with CNIs that are not ACI.

Reply:
while refactoring the code, removed this variable in [c1e3dc3](https://github.com/rancher/rke/pull/2963/commits/c1e3dc35396f7419efed3f51373f0ea613b9352d)